### ISSUE TITLE
How to get all keys root.* and transform to smthng?

### ISSUE THREAD

**Issue by s71m** [seperman/redisworks#26]
I'm glad to see you project here, can you suggest - how to get all keys and convert rows to dataframe for example?

```
class Tune(Root):
    pass

tune = Tune(**settings.redis_settings)
uid = 'ba64a3c7-dd1d-4f19-8758-94aac17d971b'
common = {
        'a': {
            'is_active': True,
            'value': 10000
        },
        'b': {
            'is_active': False,
            'value': 20000
        }
}
tune[uid] = common
for key in tune.red.keys('tune*'):
    print(key)
```

And that i get b'tune.ba64a3c7-dd1d-4f19-8758-94aac17d971b', how can i achieve to pass this key to get value? Should i just extract ba64a3c7-dd1d-4f19-8758-94aac17d971b from key string value and pass tune['ba64a3c7-dd1d-4f19-8758-94aac17d971b']

**Comment by s71m** [comment]
sorry, my fault. 
prefix b means byte string
redis.StrictRedis(host="localhost", port=6379, charset="utf-8", decode_responses=True)
it returns 'tune.ba64a3c7-dd1d-4f19-8758-94aac17d971b'

**Comment by seperman** [comment]
Cool!Sep DehpourOn Oct 11, 2023, at 1:47 PM, s71m ***@***.***> wrote:﻿
Closed #26 as completed.
